### PR TITLE
2 0 remove deleted node from perimeters

### PR DESCRIPTION
**Pull request by ngilain**
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2277
https://github.com/open-orchestra/open-orchestra-model-interface/pull/309

### CODE REVIEW

**Comment by itkg-nanne** [comment]
:+1:

File: GroupBundle/Repository/GroupRepository.php [hunk 1]
```diff
@@ -168,10 +168,30 @@ public function updatePerimeterItem($perimeterType, $oldItem, $newItem, $siteId)
...
 
+        $this->removeItemFromPerimeter($perimeterKey, $item, $siteId);
+    }
+
+    /**
+     * @param string $perimeterType
+     * @param string $item
+     * @param string $siteId
+     */
+    public function removePerimeterItem($perimeterType, $item, $siteId)
```
Comment (itkg-nanne) [inline GroupBundle/Repository/GroupRepository.php]: Should be added in GroupRepositoryInterface

File: GroupBundle/Repository/GroupRepository.php [hunk 2]
```diff
@@ -168,10 +168,30 @@ public function updatePerimeterItem($perimeterType, $oldItem, $newItem, $siteId)
...
+    {
+        $this->removeItemFromPerimeter('perimeters.' . $perimeterType . '.items', $item, $siteId);
+    }
+
+    /**
+     * @param string $perimeterKey
+     * @param string $item
+     * @param string $siteId
+     */
+    protected function removeItemFromPerimeter($perimeterKey, $item, $siteId)
```
Comment (itkg-nanne) [inline GroupBundle/Repository/GroupRepository.php]: Should be added in GroupRepositoryInterface

Comment (ngilain) [inline GroupBundle/Repository/GroupRepository.php]: @itkg-nanne : nope, this is a protected method

File: GroupBundle/Repository/GroupRepository.php (hunk 1 above)
Comment (ngilain) [inline GroupBundle/Repository/GroupRepository.php]: @itkg-nanne : this is already the case

**Comment by ngilain** [comment]
@alavieille : update done

**Pull request merged**